migration/migrator/data/course_tables.sql

```diff
@@ -13,20 +13,6 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
---
--- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: -
---
-
-CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
```

Comment:
Why did this blob of the two .sql files get deleted?

Reply:
something strange happened when I did a course dump, all fixed now.